modules/cassandrabackend/cassandrabackend.cc

```diff
@@ -0,0 +1,324 @@
+		std::string username = getArg("username");
+		std::string password = getArg("password");
+		std::string keyspace = getArg("keyspace");
+		int core_connections = getArgAsNum("core-connections");
+		int max_connections = getArgAsNum("max-connections");
+		int max_concurrent_creations = getArgAsNum("max-concurrent-creations");
+		int num_io_threads = getArgAsNum("num-io-threads");
+		int protocol_version = getArgAsNum("protocol-version");
+		int queue_size_io = getArgAsNum("queue-size-io");
+		int queue_size_event = getArgAsNum("queue-size-event");
+		int reconnect_wait_time = getArgAsNum("reconnect-wait-time");
+		int concurrent_requests_threshold = getArgAsNum("concurrent-requests-threshold");
+		int connect_timeout = getArgAsNum("connect-timeout");
+		int request_timeout = getArgAsNum("request-timeout");
+		int enable_load_balance_round_robin = getArgAsNum("enable-load-balance-round-robin");
+		int enable_token_aware_routing = getArgAsNum("enable-token-aware-routing");
+		int enable_latency_aware_routing = getArgAsNum("enable-latency-aware-routing");
+		int enable_tcp_nodelay = getArgAsNum("enable-tcp-nodelay");
+		int enable_tcp_keepalive = getArgAsNum("enable-tcp-keepalive");
+		all_authoritative_server = getArgAsNum("all-authoritative-server");
+
+		L << Logger::Info << " cassandra-seed-nodes " << seed_nodes << " cassandra-keyspace " << keyspace << endl;
+		L << Logger::Info << " cassandra-username " << username << " cassandra-password " << password << endl;
+		L << Logger::Info << " cassandra-core-connections " << core_connections << " cassandra-max_connections " << max_connections << endl;
+		L << Logger::Info << "[cassandradbmanager] cassandra-protocol-version " << protocol_version << endl;
+		L << Logger::Info << "[cassandradbmanager] cassandra-max-concurrent-creations " << max_concurrent_creations << endl;
+		L << Logger::Info << "[cassandradbmanager] cassandra-queue-size-io " << queue_size_io << " cassandra-queue-size-event " << queue_size_event << endl;
+		L << Logger::Info << "[cassandradbmanager] cassandra-reconnect-wait-time " << reconnect_wait_time << " cassandra-concurrent-requests-threshold " << concurrent_requests_threshold << endl;
+		L << Logger::Info << "[cassandradbmanager] cassandra-connect-timeout " << connect_timeout << " cassandra-request-timeout " << request_timeout << endl;
+		L << Logger::Info << "[cassandradbmanager] cassandra-enable-load-balance-round-robin " << enable_load_balance_round_robin << " cassandra-enable-token-aware-routing " << enable_token_aware_routing << endl;
+		L << Logger::Info << "[cassandradbmanager] cassandra-enable-latency-aware-routing " << enable_latency_aware_routing << endl;
+		L << Logger::Info << "[cassandradbmanager] cassandra-enable-tcp-nodelay " << enable_tcp_nodelay << " cassandra-enable-tcp-keepalive " << enable_tcp_keepalive << endl;
+		L << Logger::Info << "[cassandradbmanager] all-authoritative-server " << all_authoritative_server <<endl;
```

Comment:
Is it really necessary to log all these on every time this backend is spawned (can happen many times during application lifecycle)